src/EnergyPlus/SingleDuct.cc

```diff
@@ -6355,7 +6360,7 @@ void AirTerminalMixerData::InitATMixer(EnergyPlusData &state, bool const FirstHV
             airLoopOAFrac = state.dataAirLoop->AirLoopFlow(this->AirLoopNum).OAFrac;
             if (airLoopOAFrac > 0.0) {
                 vDotOAReq = DataSizing::calcDesignSpecificationOutdoorAir(state, this->OARequirementsPtr, this->ZoneNum, UseOccSchFlag, true);
-                mDotFromOARequirement = vDotOAReq * state.dataEnvrn->StdRhoAir / airLoopOAFrac;
+                mDotFromOARequirement = vDotOAReq * state.dataEnvrn->StdRhoAir / (airLoopOAFrac * this->ADEff);
```

Comment:
What should be done here is include ADEff inside calcDesignSpecificationOutdoorAir so other models will also get this update

Reply:
This was the change needed to see differences in the simulation results. Without this change there are only diffs in the eio.

Reply:
I don't know the best solution, but I don't think we want to be mixing extra functionality into DesignSpecification:OutdoorAir.  The issue here is that some of the info is in DesignSpecification:OutdoorAir and some is in DesignSpecification:ZoneAirDistribution.  I think these sorts of complexities should be handled at the parent level and these design spec object models should be kept simple, just the dry results please, and let the parents do what they need with the specification.

Reply:
The ATMixer receives OA from the system. There is no adjustment at the ATMixer. So maybe the system (DOAS) is not providing the correct amount of OA. I'm still looking at that. It appears the TUs do need this adjustment and I will look at that after I figure out if the ATMixer really has an issue.